source/_integrations/iammeter.markdown

```diff
@@ -13,6 +13,14 @@ ha_codeowners:
 
 `iammeter` provides real-time readings of single-phase (WEM3080, WEM3162) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
 
+  
+demo show:
+
+<p class='img'>
+  <img src='{{site_root}}/images/screenshots/iammeter demo.jpg' />
```

Comment:
do you need this `{{site_root}}` here? 

Also, Can you rename the file to be with out spaces?

Reply:
{{site_root}} removed and filename renamed.

thanks for your answer.